### ISSUE TITLE
reproducing m4a in streaming

### ISSUE THREAD

**Issue by JenGL** [goldfire/howler.js#477]
I'm downloading an m4a adio file divided in chunk every chunk is 1 or 2 Unit8Array of 256k binary data.

i'm able to reproduce the first chunk but no the others. Howler gives no error but the other chunks are not reproduced at all

**Comment by JenGL** [comment]
probably Howler is not able to decode the following chunks because it need some information that is contained only on the first chunk... it is possible to extract these information from the Unit8Array and to add these info in the other chunks ?

**Comment by goldfire** [comment]
That is correct, the metadata is stored at the beginning of the file and isn't present at all in the rest. Can you provide some sample code of a use-case by chance?

**Comment by JenGL** [comment]
playThis(streamer) {
    const play = () => {
        this._crtAudioStreamer.getNext().then((obj) => {
          const blob = new Blob(obj.new, {
            'type': this._crtAudioStreamer.metadata.data.mimeType
          });
          const sound =  new Howl({
      src: [URL.createObjectURL(blob)],
      format: this._crtAudioStreamer.metadata.data.extension,
      // html5: true,
      onend: () => {
        this._chainNext(); // put in play the next chunk
      },
      onloaderror: (data, err) => {
        console.warn("Error while trying to reproduce audio.");
        console.error(data, err);
      },
      onplay: () => {
        if (obj.others) {
          play();
        } else {
          this._gotAllChunks = true;
        }
      }
    });
          if (this._chunkInPlay) {
            this._soundChain.push(sound); // chunks to be played after this one
          } else {
            this._crtSound = sound;
            this._chunkInPlay = true;
            this._crtSound.play();
          }
        });
    };

**Comment by JenGL** [comment]
html5: ture is commented because it's not workin ! #470 

i download the file in chunks and create a new sound for each chunk, then i put each chunk in play in chian.

Is there a way to put the new chunk downloaed in the same Howl sound ? May be in this way he can decode the new chunk with the same header information

**Comment by JenGL** [comment]
i solved this problem by using fragment mp4 + MediaSourceExtension and passing to Howler the CreateUrl from the media source element, would be great to have this directly on howler

**Comment by baadc0de** [comment]
Sorry for the hijack; I've been looking at Howler for streaming too, in my case I get data over websockets. Aurora.js can be configured to accept such a stream, are there any plans to make Howler play back on demand streaming data?
